### PR TITLE
Cleaning up dead code

### PR DESCRIPTION
**Pull request by alacuku**
# Description

This PR removes dead code not anymore used after the merge of #697.

### CODE REVIEW

**Review by palexster — APPROVED** [review]

File: internal/liqonet/route-operator/symmetricRoutingOperator.go [hunk 1]
```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/liqotech/liqo/pkg/liqonet/utils"
+
```
Comment (giorio94) [inline internal/liqonet/route-operator/symmetricRoutingOperator.go:11]: Should be grouped with the others

File: pkg/liqonet/routing/vxlanRouting_test.go [hunk 2]
```diff
@@ -4,6 +4,8 @@ import (
 	"net"
 	"strconv"
 
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+
```
Comment (giorio94) [inline pkg/liqonet/routing/vxlanRouting_test.go:8]: Same here

File: deployments/liqo/values.yaml [hunk 3]
```diff
@@ -71,8 +71,6 @@ networkManager:
     # you need tell liqo the subnets used in your cluster. E.g if your cluster nodes belong to the 192.168.2.0/24 subnet then
     # you should add that subnet to the reservedSubnets. PodCIDR and serviceCIDR used in the local cluster are automatically added to the reserved list.
     reservedSubnets: []
-    # -- Set this field to true if you are deploying liqo in GKE cluster
-    GKEProvider: false
```
Comment (giorio94) [inline deployments/liqo/values.yaml:75]: This flag should also be dropped from the online documentation.

**Review by giorio94 — APPROVED** [review]
/lgtm

**Pull request merged**